ast/ddl.go

```diff
@@ -54,6 +54,12 @@ type CharsetOpt struct {
 	Col string
 }
 
+// NullString  is the non-terminal symbol
```

Comment:
```suggestion
// NullString represents a string that may be nil.
```

Reply:
NullString represents a string that may be nil ，which  is good   and  is equal to mysql class. 

 A empty index  is NullString （true），anonymous index is not nil（false） ，I will modify the code？